ydb/core/protos/auth.proto

```diff
@@ -53,6 +53,7 @@ message TAuthConfig {
     optional bool UseBuiltinDomain = 78 [default = true];
     optional string AccessServiceType = 79 [default = "Yandex_v2"]; // For now the following values are supported: "Yandex_v2", "Nebius_v1"
     optional string CertificateAuthenticationDomain = 80 [default = "cert"];
+    optional bool EnableBuiltinAuthMechanism = 81 [default = true];
```

Comment:
it's called "Login".
we bave builtin tokens, and it's a different thing...

Reply:
done